### PR TITLE
Feature/userstory 5 - Shopping Cart

### PR DESCRIPTION
**Pull request by YuriPadilla**

### CODE REVIEW

**Comment by YuriPadilla** [comment]
Displaying the page to see the products in the shopping cart will be developed in the next user story. Thanks for reviewing!!!

**Review by marcterre — APPROVED** [review]
just some naming you could change

File: components/BikesList/index.js [hunk 1]
```diff
@@ -2,27 +2,46 @@ import { bikes } from "../../lib/bikes.js";
...
+    } else {
+      isSelected = selectedProducts.some((product) => product.id === bike.id);
+    }
+    return !isSelected;
+  });
+
   return (
     <>
       <h2>Bikes</h2>
       <StyledBikesUl>
```
[the remark this answers is not in the document]
Comment (YuriPadilla) [reply components/BikesList/index.js:25]: Hi Marc!!! Thanks for your review!!!  :)  :)
- It is recommended to use the word "Styled" in front of a name to name all the styled elements,
- I use at the end of the name the word `ul` to realize easier that it is about an `ul` list. It's better so to read the code.

File: components/BikesList/index.js [hunk 2]
```diff
@@ -2,27 +2,46 @@ import { bikes } from "../../lib/bikes.js";
...
-                  priority
-                />
-              </Link>
-            </StyledBikeLi>
-          );
-        })}
+        {bikesToShow.length > 0 ? (
+          bikesToShow.map((bike) => {
+            return (
+              <StyledBikeLi key={bike.id}>
```
[the remark this answers is not in the document]
Comment (YuriPadilla) [reply components/BikesList/index.js:29]: The same here:
- The name "StyledBikeListItem" is too long, and...
- between "ListItem" and "Li" I choose "Li", I mean: "StyledBikeLi" and not "StyledBikeListItem". And also so I can realize that it is about a `li`. It is better so... when reading a code.

File: components/ShoppingCart/ShoppingCart.styled.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  border: none;
+  border-radius: 50%;
+  background: transparent;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 2px 3px 20px rgb(95, 117, 129);
+`;
+
+export const StyledQuantityDiv = styled.div`
```
[the remark this answers is not in the document]
Comment (YuriPadilla) [reply components/ShoppingCart/ShoppingCart.styled.js:20]: That about "Wrapper"... I like it...!!!  😀 👍
...I will use it. Thanks!!!

File: components/ShoppingCart/ShoppingCart.styled.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export const StyledShoppingCart = styled(Link)`
+  position: fixed;
+  z-index: 1;
+  top: 85px;
+  right: 20px;
+  width: 70px;
```
[the remark this answers is not in the document]
Comment (YuriPadilla) [reply components/ShoppingCart/ShoppingCart.styled.js:9]: Sehr gut!!! Thanks!!! I will do it so as you mean... more responsive... but I want to do all that later when I style the whole WebApp.
Thanks, Marc!!! 😀 👍

**Pull request merged**